### ISSUE TITLE
Doesn't seem to work with ESM builds for nodejs env

### ISSUE THREAD

**Issue by amalitsky** [sindresorhus/gzip-size#23]
Since it is using [deprecated Duplexer library](https://github.com/Raynos/duplexer), (with [15M weekly downloads btw](https://www.npmjs.com/package/duplexer)) which has only CJS build to it.

Building my package with `tsup` (goes through fine) and trying to run the "binary", I get the following error message:
```
throw Error('Dynamic require of "' + x + '" is not supported');
        ^
Error: Dynamic require of "stream" is not supported
```

First line of duplexer's build is:
```ts
var Stream = require("stream")
```
And it is causing problems. 

My ts config uses `NodeNext` as `module` and `moduleResolution`, and I have `"type": "module"` in package.json.

Once I drop `gzip-size` import from my code, all builds and works fine.

Let me know how I can help and thank you for the library! :heart: 

Seemingly [related issue](https://github.com/evanw/esbuild/issues/1921) with _esbuild_. I use _tsup_ but these are similar tools.

**Comment by amalitsky** [comment]
Added [repository with reproduction](https://github.com/amalitsky/gzip-size-import-issue) of an issue I'm experiencing.

**Comment by sindresorhus** [comment]
No, it's using the Node.js API. The problem is your build config, not this package.
